contrib/remote-ssh-build.bash

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# vim: softtabstop=2 shiftwidth=2 expandtab
+
+## REMOTE SSH ACCESS
+
+# This script builds a zbm with remote access support via ssh. The image and
+# efi executable will contain private host keys and public keys used for
+# mutual authentication.
+
+# All keys / settings are placed into ${RS_DIR} ('ssh-data' in the current
+# working directory by default). Its contents can be customized as needed.
+# They are regenerated if missing.
+
+# Server private keys are generated automatically as needed. It is
+# recommended to not reuse existing private keys.
+# User authentication for the root account is done via 'authorized_keys'.
+# The current users' authorized keys are copied if it is not present.
+
+RS_DIR=$(realpath "${RS_DIR:-ssh-data}")
+RS_KEYDIR="${RS_DIR}/keys"
```

Comment:
Calling this `dropbear` is a bit more descriptive.
```suggestion
RS_KEYDIR="${RS_DIR}/dropbear"
```

Reply:
I think the fact that we use dropbear inside might not be super relevant for users, but that the directory contains the keys is important.